joint_trajectory_controller/include/joint_trajectory_controller/tolerances.hpp

```diff
@@ -92,7 +92,7 @@ struct SegmentTolerances
  * \return Trajectory segment tolerances.
  */
 SegmentTolerances get_segment_tolerances(
-  const rclcpp::Node & node, const std::vector<std::string> & joint_names)
+  const rclcpp_lifecycle::LifecycleNode & node, const std::vector<std::string> & joint_names)
```

Comment:
We should probably here use SharedPtr to the node and not the object itself. This is usually done in the rest of the code-base.